src/scripts/main.js

```diff
@@ -1,3 +1,16 @@
 'use strict';
-
 // write your code here
+
+const populations = document.querySelectorAll('.population');
+const totalPopulation = document.querySelector('.total-population');
+const totalAverage = document.querySelector('.average-population');
+const populationSum = [...populations]
+  .map(item => item.innerText.split(',').join(''))
```

Comment:
do u really need to create a copy before `map`?

Reply:
Yep, querrySelectorAll returns a pseudo array whose __proto__ property not containing array methods, that's why I before using map() convert it to array by destructuring